cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -73,15 +73,7 @@ Unix socket-based connections.`)
 		`Open sockets for each Cloud SQL Instance in the projects specified
 (comma-separated list)`,
 	)
-	instances = flag.String("instances", "",
```

Comment:
Why was the instances flag moved to the runProxy() function?

Reply:
It's a side effect of how `flag.Var` works in effect. We need a type that implements the `Value` interface, so we can override the `Set` behavior. Since `flag.Var` doesn't return a value, moving the definition was a reasonable approach.


https://pkg.go.dev/flag#Var

Reply:
Thanks for the clarification, Eno!